install/kubernetes/helm/istio/README.md

```diff
@@ -112,7 +112,7 @@ The chart deploys pods that consume minimum resources as specified in the resour
     type: Opaque
     data:
       username: YWRtaW4=
-      passphrase: MWYyZDFlMmU2N2Rm
```

Comment:
I think this is right, you may see for Grafana, we are using `passphrase` for password in `values.yaml` at https://github.com/istio/istio/blob/master/install/kubernetes/helm/subcharts/grafana/values.yaml#L16 , am I missing anything?

Reply:
tend to agree.  Is there something broken with the existing implementation?

Reply:
When I tried to create the secret using `passphrase` I got an error saying the `password` key was missing.
Beside, the `secret.yaml` under `istio-1.0.5/install/kubernetes/helm/istio/charts/grafana/templates` shows `password`:

```
{{- if .Values.security.enabled -}}
apiVersion: v1
kind: Secret
metadata:
  name: grafana
  namespace: {{ .Release.Namespace }}
  labels:
    app: grafana
type: Opaque
data:
  username: {{ .Values.security.adminUser | b64enc | quote }}
  password: {{ .Values.security.adminPassword | b64enc | quote }}
{{- end -}}
```

Finally, deployment file expect the key `password` as well:

```
- name: GF_SECURITY_ADMIN_PASSWORD
  valueFrom:
    secretKeyRef:
      name: grafana
      key: password
```

Reply:
Any chance for the `values.yaml` file you guys are pointed out to be wrong as well?

Reply:
@gottfrois I see, you are using Istio 1.0.5 which does not include the fix https://github.com/istio/istio/pull/7719 . For master branch, the `secret.yaml` was already deleted and end user need to create the secret first before they install Istio as we do not want to expose username and password in `values.yaml`.

Reply:
Makes sense then 👍

Reply:
@gottfrois can you help close this PR if no further action needed?